### DISCUSSION TITLE
Issues with Russian language transcription

### DISCUSSION THREAD

**Discussion by BayramAnnakov** [openai/whisper#1391] [General]
Some random text appears when transcribing from Russian - see examples below
<img width="741" alt="Screenshot 2023-05-25 at 3 26 11 PM" src="https://github.com/openai/whisper/assets/774131/69cdc3f0-8b02-4298-91fd-dd859a74fa77">

this repeated phrase never appears in the audio; probably, this is because of silences

**Comment by phineas-pta** [comment]
it's called hallucination and not unique to russian see #928, for possible solutions see #679
